binding_generator.py

```diff
@@ -2745,12 +2745,12 @@ def correct_typed_dictionary(type_name):
 def correct_type(type_name, meta=None, use_alias=True):
     type_conversion = {"float": "double", "int": "int64_t", "Nil": "Variant"}
     if meta is not None:
```

Comment:
What if meta is not `None`, but we don't handle it here (unexpected value)?

Reply:
It will ignore it. That's actually the whole point of this PR: if there's an unexpected value in "meta", then we'll ignore it and not potentially generate invalid code.

PR https://github.com/godotengine/godot/pull/86079 adds "required" as a potential value for "meta", and without this PR, it'll generate invalid code (attempting to use "required" as a type name).